include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -1171,11 +1171,19 @@ class FEPointEvaluation
    * @param first_selected_component For multi-component FiniteElement
    * objects, this parameter allows to select a range of `n_components`
    * components starting from this parameter.
+   *
+   * @param lexicographic_numbering If this class is used together with
+   * FEEvaluation this parameter should be set to 'true' because FEEvaluation
+   * stores the DoF values internally always in lexicographic numbering.
```

Comment:
I think this statement could be extended somewhat because the vector entries to be interpolated could come from a third source, besides `FEEvaluation` or the conventional `DoFAccessor::get_dof_values`. I would therefore write this as:
```suggestion
   * @param force_lexicographic_numbering By default, this classes uses the native
   * numbering of degrees of freedom by the underlying FiniteElement object.
   * However, it is possible to force a lexicographic ordering of DoFs instead
   * by setting this parameter to true, which is useful if this class is used together with
   * FEEvaluation, for instance. This is because DoF values within FEEvaluation
   * are stored in lexicographic numbering.
```
Please also note that I would write this as `force_lexicographic_ordering` to make sure that we override the numbering of the finite element (as you state in the next sentence below).

Reply:
Yes, I think `force_lexicographic_ordering` makes sense! I have also adjusted the parameter documentation based on your suggestion. I kept the links to `FE_Q` and `FE_DGQ` as they are helpful for users who want to read up on hierarchic vs. lexicographic DoF numbering.